meshroom/ui/qml/GraphEditor/AttributeItemDelegate.qml

```diff
@@ -184,6 +191,121 @@ RowLayout {
             }
         }
 
+        Component {
+            id: textArea_component
+
+            Rectangle {
+                // Fixed background for the flickable object
+                color: palette.base
+                width: parent.width
+                height: 70
+
+                Flickable {
+                    width: parent.width
+                    height: parent.height
+                    contentWidth: width
+                    contentHeight: height
+
+                    ScrollBar.vertical: ScrollBar {
+                        policy: ScrollBar.AlwaysOn
+                    }
+
+                    TextArea.flickable: TextArea {
+                        wrapMode: Text.WordWrap
+                        padding: 0
+                        rightPadding: 5
+                        bottomPadding: 2
+                        topPadding: 2
+                        readOnly: !root.editable
+                        onEditingFinished: setTextFieldAttribute(text)
+                        text: attribute.value
+                        selectByMouse: true
+                        onPressed: {
+                            root.forceActiveFocus()
+                        }
+                        Component.onDestruction: {
+                            if(activeFocus)
+                                setTextFieldAttribute(text)
+                        }
+                        DropArea {
+                            enabled: root.editable
+                            anchors.fill: parent
+                            onDropped: {
+                                if(drop.hasUrls)
+                                    setTextFieldAttribute(Filepath.urlToString(drop.urls[0]))
+                                else if(drop.hasText && drop.text != '')
+                                    setTextFieldAttribute(drop.text)
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+        Component {
+            id: color_textField_component
+            RowLayout {
+                CheckBox {
+                    id: color_textField_checkbox
+                    Layout.alignment: Qt.AlignLeft
+                    checked: node.color === "" ? false : true
+                    text: checked ? "Disable custom color" : "Enable custom color"
```

Comment:
```suggestion
                    text: "Custom Color"
```
Avoid ambiguity in the name (the status or the action) and use Camel Case.

Reply:
Done.